x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/mocks.ts

```diff
@@ -122,37 +124,66 @@ export const fleetGetPackageListHttpMock = httpHandlerMockFactory<FleetGetPackag
 export type FleetGetAgentPolicyListHttpMockInterface = ResponseProvidersInterface<{
   agentPolicy: () => GetAgentPoliciesResponse;
 }>;
-export const fleetGetAgentPolicyListHttpMock = httpHandlerMockFactory([
-  {
-    id: 'agentPolicy',
-    path: AGENT_POLICY_API_ROUTES.LIST_PATTERN,
-    method: 'get',
-    handler: () => {
-      const generator = new EndpointDocGenerator('seed');
-      const endpointMetadata = generator.generateHostMetadata();
-      const agentPolicy = generator.generateAgentPolicy();
-
-      // Make sure that the Agent policy returned from the API has the Integration Policy ID that
-      // the endpoint metadata is using. This is needed especially when testing the Endpoint Details
-      // flyout where certain actions might be disabled if we know the endpoint integration policy no
-      // longer exists.
-      (agentPolicy.package_policies as string[]).push(endpointMetadata.Endpoint.policy.applied.id);
-
-      return {
-        items: [agentPolicy],
-        perPage: 10,
-        total: 1,
-        page: 1,
-      };
+export const fleetGetAgentPolicyListHttpMock = httpHandlerMockFactory<FleetGetAgentPolicyListHttpMockInterface>(
```

Comment:
the highlighted changes to this function is just do to indentation after I added this:
```
httpHandlerMockFactory<FleetGetAgentPolicyListHttpMockInterface>
```